components/ui-api-layer/README.md

```diff
@@ -97,3 +97,15 @@ KUBE_CONFIG=/Users/{your_username}/.kube/config go test ./... -tags=acceptance
 ### Verify the code
 
 To check if the code is correct and you can push it, run the `before-commit.sh` script. It builds the application, runs tests, checks the status of the vendored libraries, runs the static code analysis, and ensures that the formatting of the code is correct.
+
+
+### Custom UsageKind
+
+In order to specify a new custom UsageKind, you have to adjust RBAC roles of the UI API Layer. UsageKind contains a reference to the resource. You have to give UI API Layer `get`, `list` and `watch` permissions to this resource.
+If UsageKind references to the Pods, then you have to add below permission to the UI API Layer RBAC [roles](../../resources/core/charts/ui-api/templates/cluster-role.yaml):
```

Comment:
1. ~references~ -> `refers`
2. ~then~
3. ~have to~ -> `must`
4. ~below~

Add this sentence at the end:
`See the following example:`